detectors/node/opentelemetry-resource-detector-aws/src/detectors/AwsLambdaDetectorSync.ts

```diff
@@ -19,47 +19,32 @@ import {
   IResource,
   Resource,
   ResourceAttributes,
-  ResourceDetectionConfig,
 } from '@opentelemetry/resources';
-import {
-  SEMRESATTRS_CLOUD_PROVIDER,
-  SEMRESATTRS_CLOUD_PLATFORM,
-  SEMRESATTRS_CLOUD_REGION,
-  SEMRESATTRS_FAAS_VERSION,
-  SEMRESATTRS_FAAS_NAME,
-  CLOUDPROVIDERVALUES_AWS,
-  CLOUDPLATFORMVALUES_AWS_LAMBDA,
-} from '@opentelemetry/semantic-conventions';
 
 /**
  * The AwsLambdaDetector can be used to detect if a process is running in AWS Lambda
  * and return a {@link Resource} populated with data about the environment.
  * Returns an empty Resource if detection fails.
  */
 export class AwsLambdaDetectorSync implements DetectorSync {
-  detect(_config?: ResourceDetectionConfig): IResource {
```

Comment:
Removal of this config parameter is a breaking change ([see other usage](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/resource-detector-aws-v1.9.0/detectors/node/opentelemetry-resource-detector-aws/src/detectors/AwsLambdaDetector.ts#L32)). While not used here, it could be useful later, so let's keep it.